tests.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+set -e
+
+help=$(cat << EOF
+Launch tests, linters or coverage for Hermes
+
+Linters:
+./tests.sh linters  # launch all linters listed below
```

Comment:
Is it actually feasible to have Pylint and Flake8 at once?

Reply:
While they are quite similar, I don't find it harmful to use both of them (until they're conflicting, which I haven't faced yet), as they sometimes provide different functionality